openedx_learning/core/components/admin.py

```diff
@@ -1,34 +1,39 @@
+"""
+Django admin for components models
+"""
 from django.contrib import admin
 from django.template.defaultfilters import filesizeformat
 from django.urls import reverse
 from django.utils.html import format_html
+from django.utils.safestring import SafeText
 
+from openedx_learning.lib.admin_utils import ReadOnlyModelAdmin
 from .models import (
     Component,
     ComponentVersion,
     ComponentVersionRawContent,
 )
-from openedx_learning.lib.admin_utils import ReadOnlyModelAdmin
 
 
 class ComponentVersionInline(admin.TabularInline):
+    """Inline admin view of ComponentVersion from the Component Admin"""
```

Comment:
Nit: Our [python](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings) guidelines state that we should:
> "Always format docstrings using the multi-line convention, even if there’s only one line of content".

I saw that you're fixing a lot of functions without docstring here. Is it a good time to fix this (at least in the changed ones)?

Reply:
Oh, I didn't know that was the official guideline. That's actually my personal preference too; I just used the one line form because it was already in use in this repo. I agree it makes sense to fix it now.